internal/codegen/golang/driver.go

```diff
@@ -35,7 +39,18 @@ func (d SQLDriver) Package() string {
 		return SQLPackagePGXV4
 	case SQLDriverPGXV5:
 		return SQLPackagePGXV5
+	case SQLDriverGoSqlDriverMySQL:
+		return SQLPackageGoSqlDriverMySQL
 	default:
 		return SQLPackageStandard
 	}
 }
+
+func (d SQLDriver) UsesStandard() bool {
+	switch d {
+	case SQLDriverPGXV4, SQLDriverPGXV5:
+		return false
+	default:
+		return true
+	}
+}
```

Comment:
Can we move this to another PR? This appears to be unused in this PR.

Reply:
Good call, I didn't even need it in the next PR. Removed.